src/assets/scss/_mixins.scss

```diff
@@ -28,9 +28,10 @@
 }
 
 @mixin clearfix {
-  &::after {
-    clear: both;
+  &::before, &::after {
```

Comment:
I was looking at the Bootstrap clearfix and realized that this mixin was missing `::before` here.